upgrader/apps/upgrader/lib/schemaIncrementTasks/SchemaIncrementTask66.php

```diff
@@ -30,11 +30,7 @@ public function loadSql()
 
         $sql[] = 'ALTER TABLE `hs_hr_emp_member_detail` DROP PRIMARY KEY;';
 
-        $sql[] = 'ALTER TABLE `hs_hr_emp_member_detail` ADD `id` int(6);';
-
-        $sql[] = 'ALTER TABLE `hs_hr_emp_member_detail` ADD PRIMARY KEY(`id`);';
-
-        $sql[] = 'ALTER TABLE `hs_hr_emp_member_detail` MODIFY COLUMN id INT auto_increment;';
+        $sql[] = 'ALTER TABLE `hs_hr_emp_member_detail` ADD `id` int(6) NOT NULL PRIMARY KEY AUTO_INCREMENT;';
```

Comment:
What if check 
ALTER TABLE hs_hr_emp_member_detail ADD COLUMN IF NOT EXISTS ..

Reply:
Actually the `id` column is not there before this increment task.

Reply:
Oops. MySQL not supporting this syntax in our supporting range.

Reply:
@RajithaKumara What's going to do here?

Reply:
Current state of PR working fine. If we add `ADD COLUMN IF NOT EXISTS`, query will fail in some MySQL and MariaDB versions which are in our supporting range.
Next thing is before coming to this SQL upgrader task, it fails at [this SQL query](https://github.com/orangehrm/orangehrm/blob/master/upgrader/apps/upgrader/lib/schemaIncrementTasks/SchemaIncrementTask66.php#L27) if this task runs twice.
It seems to check column existence is not required, since we have to deal with too much thing to made compatible for all supporting databases.